### ISSUE TITLE
Enable user to change his password

### ISSUE THREAD

**Issue by htunlogic** [hudikhq/hoodik#4]
User should be able to change his password.

In order to do that, user must be logged in with unlocked private key. When the password is changed, the key locked again with the new password must be sent with the new password and the user object will be updated with new credentials.

In case user forgets his password and wants to reset it, he can be offered to login with private key directly, and then he can be sent to the same screen where he will be able to change his password.

Private key signature needs to be verified before its being locked and swapped.

All the files and folders will stay as they were, and nothing else should change.

In case user loses his private key and password, there is nothing we can do, all the files will remain locked forever.

TLDR: User must have either valid password or private key to access his account
